client/app/src/js/services/services.js

```diff
@@ -66,6 +66,29 @@ function Screenshots($http, $location, $window) {
 };
 Screenshots.$inject = ['$http','$location','$window'];
 
+function UserInfo($http){
```

Comment:
The purpose of UserInfo factory is unclear. Is it to abstract the details of pulling down the user info from the server? If so, I suggest calling it `User`. And rather than creating a single UserInfo that gets passed around (not exactly show how that'd work?), I propose instantiating a `new User(username)` and pass in the username, then we'd get back the User model from the DB.

Run it by Jackson and get some input from him as well, let me know what you two think.